cmd/thanos/compact.go

```diff
@@ -449,6 +465,7 @@ func runCompact(
 type compactConfig struct {
 	haltOnError                                    bool
 	acceptMalformedIndex                           bool
+	addHashes                                      bool
```

Comment:
I am a fan of avoiding bool options. What about `hashFunction` setting when you set what kind of hash to use (: And by default it will be none?